### ISSUE TITLE
About the pre-trained diffusion model

### ISSUE THREAD

**Issue by Inn1917** [Gorilla-Lab-SCUT/Fantasia3D#8]
Great work! 
I would like to know about some details about the pre-trained diffusion model. To my understanding, the paper used a diffusion model conditioned on both normal map and text prompt to optimize the high-frequency detail of the geometry surface. Is the model trained by your own dataset or publicly available?
Thank you!

**Comment by ARuiChen** [comment]
We only used the most common stable diffusion model directly without additional training or using other custom models. To the best of our knowledge, we are the first to use the normal map as the input of the ordinary stable diffusion model.

**Comment by ARuiChen** [comment]
As for why inputting normal is also useful, I want to attribute it to the potential and powerful capabilities of stable diffusion that have not been discovered before.

**Comment by Inn1917** [comment]
Great thanks for the reply!
So the encoding is done by directly adding noise to normal map and use it as input of diffusion model to predict the noise, as the single iteration step in ordinary DM generation?
If so that would be quite interesting.

**Comment by ARuiChen** [comment]
Yes, I also think it's very interesting.

**Comment by Inn1917** [comment]
That is all I have then
Thanks again for the answers!

**Comment by Tonniia** [comment]
Hi! I would like to know the motivation for design 
`guidance_inp = torch.cat(
                    [out["comp_normal"] * 2.0 - 1.0, out["opacity"]], dim=-1)`
as latent code for stable diffusion. Thank you!
